exercises/circular-buffer/example.js

```diff
@@ -1,25 +1,28 @@
 let buffer, bufferMax;
 
-const BufferEmptyException = () => ({
-    name: "buffer empty exception!",
-    message: "can't read from an empty buffer!"
-  });
-
-const BufferFullException = () => ({
-    name: "buffer full exception!",
-    message: "can't write to a full buffer!"
-  });
+export class BufferEmptyException extends Error {
```

Comment:
On second thought, should this be `BufferEmptyException` or `BufferEmptyError` (since it inherits from `Error`)?